modules/140-user-authz/docs/README.md

```diff
@@ -25,13 +26,15 @@ In addition to the RBAC, you can use a set of high-level roles in the module:
 - `Admin` — the same as `Editor` + can delete service objects (auxiliary resources such as `ReplicaSet`, `certmanager.k8s.io/challenges` and `certmanager.k8s.io/orders`);
 - `ClusterEditor` — the same as `Editor` + can manage a limited set of `cluster-wide` objects that can be used in application tasks (`ClusterXXXMetric`, `KeepalivedInstance`, `DaemonSet`, etc.). This role is best suited for cluster operators.
 - `ClusterAdmin` — the same as both `ClusterEditor` and `Admin` + can manage `cluster-wide` service objects (e.g.,  `MachineSets`, `Machines`, `OpenstackInstanceClasses`..., as well as `ClusterAuthorizationRule`, `ClusterRoleBindings` and `ClusterRole`). This role is best suited for cluster administrators. **Note** that since `ClusterAdmin` can edit `ClusterRoleBindings`, he can **broaden his privileges within the cluster**;
-- `SuperAdmin` — can perform any actions with any objects (note that [`limitNamespaces`](#module-features) restrictions remain valid).
+- `SuperAdmin` — can perform any actions with any objects (note that [`namespaceSelector`] and [`limitNamespaces`](#module-features) restrictions remain valid).
 
 ## Implementation nuances
 
 > **Caution!** Currently, the multi-tenancy mode (namespace-based authorization) is implemented according to a temporary scheme and **isn't guaranteed to be entirely safe and secure**!
 
-The `allowAccessToSystemNamespaces` and `limitNamespaces` options in the CR will no longer be applied if the authorization system's webhook is unavailable for some reason. As a result, users will have access to all namespaces. After the webhook availability is restored, the options will become relevant again.
+If a [`ClusterAuthorizationRule`](cr.html#clusterauthorizationrule) Custom Resource contains both `limitNamespaces` and `namespaceSelector` parameters, only the latter will be taken into consideration.
```

Comment:
What about `namespaceSelector` and `allowAccessToSystemNamespaces` specified together?